docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -38,23 +38,22 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
 <h2>Release notes: Open GEE 5.2.5</h2>
 
-<p>Open GEE 5.2.5 is currently in development</p>
-
   <h5>
     <p id="overview_5.2.5">New Features</p>
   </h5>
-    <p><strong>When a GEE server is queried for server definitions the response can be returned in proper JSON format.</strong> The <code>v</code> parameter can be added to query string to specify which version of the API will be returned. The valid options are <code>v=1</code> and <code>v=2</code>. Version one returns data in the current format. Version two returns data in JSON format. An example url which returns data in JSON format looks like: <code>http://&lt;GEE earth server&gt;/&lt;database_publish_point&gt;/query?request=Json&amp;v=2</code>.</p>
+    <p><strong>When a GEE server is queried for server definitions, the response can be returned in valid JSON format.</strong> The <code>v</code> parameter can be added to the query string to specify which version of the API to use. The options are <code>v=1</code> and <code>v=2</code>. Version <code>1</code> returns data in the current format. Version <code>2</code> returns valid JSON data. An example URL that returns data in JSON format is: <code>http://&lt;GEE earth server&gt;/&lt;database_publish_point&gt;/query?request=Json&amp;v=2</code>.</p>
```

Comment:
would it be possible to see how this html would looks like ? using github doc.

Reply:
![image](https://user-images.githubusercontent.com/34994386/51635896-e18ee480-1f25-11e9-9370-a324bf3876e9.png)

Reply:
could you post the url to github docs ?

Reply:
https://www.opengee.org/geedocs/5.2.5/answer/7160006.html